README.md

```diff
@@ -65,6 +65,13 @@ python3 -u scripts/build_packages.py
 
 Freight supports the ability to have multiple versions of a package in a single Debian package archive. If a particular version of a package already exists in the Freight library, it will not be readded or overwritten.
 
+## Build peach-go-sbot debian package
+
+`sudo python3 scripts/build_peach_go_sbot.py -v <version_number>`
+
+This build the peach-go-sbot package using the latest code from go-ssb, along with a systemd unit file,
```

Comment:
Small typo here: "This build ..." -> "This builds...".

Please capitalize Debian and Freight so we can keep uniformity in the docs 🙏🏻 It's such a silly thing but is somehow highly satisfying to my brain.